apps/web/app/queries/leather-query-provider.tsx

```diff
@@ -0,0 +1,97 @@
+import { type ReactNode, createContext, useContext, useMemo } from 'react';
+
+import { ChainId } from '@stacks/network';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+
+import { NetworkConfiguration, NetworkModes } from '@leather.io/models';
+import type { RemoteConfig } from '@leather.io/query';
+
+export interface LeatherEnvironment {
+  env: string;
+  github: {
+    org: string;
+    repo: string;
+    branchName?: string;
+    localConfig?: RemoteConfig;
+  };
```

Comment:
Now that we're using these directly, do we still need the provider at all?

Reply:
hm, yeah, we can remove it